### ISSUE TITLE
Start Button not recognized by moonlight (Logitech F710 Wireless Controller)

### ISSUE THREAD

**Issue by maretodoric** [moonlight-stream/moonlight-embedded#758]
Please provide the following info.

NVidia Geforce Experience version: 3.20.0.118
Moonlight Embedded version: Moonlight Embedded 2.4.10
Moonlight Embedded source: Source
Moonlight Embedded running on: Raspberry Pi 3B+
Moonlight Embedded running on distribution: Raspbian (RetroPie)

Verbose output `-verbose` of Moonlight Embedded:
```
Moonlight Embedded 2.4.10 (PI;SDL;X11;ALSA;PULSE;EMBEDDED)
Connect to 192.168.0.100...
NVIDIA GeForce GTX 1060 6GB, GFE 3.20.0.118 (gs_04_15_26867986, 7.1.415.0)
Platform Raspberry Pi (Broadcom)
Loading mappingfile /opt/retropie/supplementary/moonlight/share/moonlight/gamecontrollerdb.txt
Add input /dev/input/by-id/usb-Logitech_Wireless_Gamepad_F710_8DFA9604-event-joystick...
Detected Logitech Gamepad F710 (030000006d0400001fc2000005030000) on /dev/input/by-id/usb-Logitech_Wireless_Gamepad_F710_8DFA9604-event-joystick as Logitech F710 Gamepad (XInput)
Add input /dev/input/by-id/usb-ShanWan_PC_PS3_Android-event-joystick...
Detected ShanWan PC/PS3/Android (03000000632500007505000010010000) on /dev/input/by-id/usb-ShanWan_PC_PS3_Android-event-joystick as SHANWAN PS3/PC Gamepad
Stream 1280 x 720, 60 fps, 20000 kbps
Initializing platform...done
Resolving host name...done
Starting RTSP handshake...done
Initializing control stream...done
Initializing video stream...done
Initializing audio stream...done
Initializing input stream...done
Starting control stream...done
Starting video stream...done
Starting audio stream...done
Starting input stream...done
ENet disconnect event pending
Server notified termination reason: 0x0100
Disconnect event timeout expired
Stopping input stream...done
Stopping audio stream...done
Stopping video stream...done
Stopping control stream...done
Cleaning up input stream...done
Cleaning up audio stream...done
Cleaning up video stream...done
Cleaning up control stream...done
Cleaning up platform...done
Sending app quit request ...
```
What is the expected result?
Start button should be recognized without issues.

What happens instead of that?
Start button is completely ignored as if not pressed.

Basically, i am unable to quit stream with BACK+START+LB+RB since moonlight cannot see that start button is pressed. Beside that, Start is also not recognized within games or within steam big picture controller identification.
I've had the same version of moonlight-embedded on previous OS installation and desided to switch to RetroPie package, when I've reinstalled from RetroPie package, i started having issues but the Start button works completely fine outside of moonlight for example Kodi, EmulationStation, other games from RetroPie, even SteamLink...

**Comment by maretodoric** [comment]
Got it resolved by editing `/etc/modprobe.d/xpad.conf` and changing `triggers_to_buttons` to `0`.
Looks like RetroPie ships with xpad driver which i didn't have in previous installation.
